package.json

```
@@ -10,13 +10,21 @@
 		"lint-check": "eslint --config .eslintrc.js . && prettier --config prettier.config.js --check **/*.{js,jsx,json,ts,tsx,md,mdx,css,html,yml,yaml,scss}"
 	},
 	"dependencies": {
+		"@react-native-async-storage/async-storage": "^1.15.8",
+		"@tensorflow-models/coco-ssd": "^2.2.2",
 		"@tensorflow/tfjs": "^3.9.0",
+		"@tensorflow/tfjs-react-native": "^0.7.0",
 		"expo": "~42.0.1",
 		"expo-camera": "~11.2.2",
+		"expo-gl": "^10.4.2",
+		"expo-gl-cpp": "^10.4.1",
+		"expo-image-manipulator": "^9.2.2",
 		"expo-status-bar": "~1.0.4",
+		"jpeg-js": "^0.4.3",
 		"react": "16.13.1",
 		"react-dom": "16.13.1",
 		"react-native": "https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz",
+		"react-native-fs": "^2.18.0",
 		"react-native-web": "~0.13.12"
```

```suggestion
		"@tensorflow-models/coco-ssd": "^2.2.2",
		"@tensorflow/tfjs": "^3.9.0",
		"@tensorflow/tfjs-react-native": "^0.7.0",
		"expo": "~42.0.1",
		"expo-camera": "~11.2.2",
		"expo-status-bar": "~1.0.4",
		"jpeg-js": "^0.4.3",
		"react": "16.13.1",
		"react-dom": "16.13.1",
		"react-native": "https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz",
		"react-native-web": "~0.13.12"
```
some of these look like they are not being used, but feel free to correct me if I am wrong

---

I'll see if I can narrow it down a bit

---

We are probably going to want `expo-image-manipulator` later for image resizing, but I've removed it for now. All the rest are necessary.